gist/src/main/java/build/gist/data/listeners/Queue.kt

```diff
@@ -149,7 +151,12 @@ class Queue: GistListener {
     }
 
     override fun onMessageShown(message: Message) {
-        logView(message)
+        val gistProperties = GistMessageProperties.getGistProperties(message)
+        if (gistProperties.persistent) {
+            Log.i(GIST_TAG, "Persistent message shown, skipping logging view")
```

Comment:
Should we include the message id in this log message like we do for the others? 

```suggestion
            Log.i(GIST_TAG, "Persistent message shown: ${message.messageId}, skipping logging view")
```